src/SocialAuthUserManager.php

```diff
@@ -319,11 +431,13 @@ public function createUser($name, $email) {
     $violations = $new_user->validate();
     if (count($violations) > 0) {
       $msg = $violations[0]->getMessage();
-      drupal_set_message($this->t('Creation of user account failed: @message', array('@message' => $msg)), 'error');
-      $this->loggerFactory
-        ->get($this->getPluginId())
-        ->error('Could not create new user: @message', array('@message' => $msg));
-      return FALSE;
+      if($msg<>'Email field is required.') {
```

Comment:
Please remove this statement and the if block after it. It is not necessary to validate here.